MAUI/Calendar/selections.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: post
+title: Selections in MAUI Calendar widget | Syncfusion
+description: Learn here all about Selections feature in Syncfusion MAUI Calendar (SfCalendar) widget and more.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Selections in MAUI Calendar (SfCalendar)
+The Selection mode is specified in the calendar property enumeration. Can select the dates by tap the cell in the calendar. `SfCalendar` provides three types mode such as `Single`, `Multiple` and `Range` Selection. The default `SelectionMode` is Single that allows the user to select one date at a time.
+
+>**NOTE** Can able to select the cells in the Year, Decade and Century views in the calendar only when the `AllowViewNavigation` property is set to false.
+
+## Single selection
+The `Single` selection can be performed in the Calendar by setting the `CalendarSelectionMode` property to `Single`. In this Selection, you can able to select a single date from the cell.
```

Comment:
The `Single` selection can be performed in the Calendar by setting the `CalendarSelectionMode` property to `Single.` In this selection, you can select a single date from the cell.

Reply:
Content modified